core/store/orm/config.go

```diff
@@ -767,12 +767,6 @@ func (c Config) JobPipelineResultWriteQueueDepth() uint64 {
 	return c.getWithFallback("JobPipelineResultWriteQueueDepth", parseUint64).(uint64)
 }
 
-// JobPipelineParallelism controls how many workers the pipeline.Runner
-// uses in parallel (how many pipeline runs may simultaneously be executing)
-func (c Config) JobPipelineParallelism() uint8 {
```

Comment:
Yeah fair concern. Right now the run execution is constrained by an upper bound of jobs*(#parallel paths per job graph) because of the way ExecuteRun is implemented - and on average it would be much less than that upper bound because the execution would be staggered. Seems fine to me, if it became a problem we could certainly bring back this env var and restructure ExecuteRun around it